.circleci/config.yml

```diff
@@ -33,29 +33,21 @@ jobs:
             python3 -m venv venv
             . venv/bin/activate
             python -m pip install --upgrade pip
-            pip install -r requirements-dev.txt
-            python setup.py install
-
 
       - save_cache:
           paths:
             - ./venv
-          key: v1-dependencies-{{ checksum "requirements-dev.txt" }}
+          key: v1-dependencies-{{ checksum "pyproject.toml" }}-{{ checksum "setup.cfg" }}-{{ checksum "setup.py" }}
 
-      # run tests!
-      # this example uses Django's built-in test-runner
-      # other common Python testing frameworks include pytest and nose
-      # https://pytest.org
-      # https://nose.readthedocs.io
       - run:
           name: run tests
           command: |
             . venv/bin/activate
-            mkdir test-reports
-            py.test --junitxml=test-reports/junit.xml tests/
+            python -m pip install .[dev]
+            python -m pytest
```

Comment:
Do we still need the CircleCI thing since now the codecov option is working properly ? 
We might just now delete all reference to CircleCI, right ?

Reply:
yes you are right.

Reply:
I will see how deleting impact the whole thing !

Reply:
@deepcharles 
Can you click in "Stop Building" here on CircleCI ? https://app.circleci.com/settings/project/github/deepcharles/ruptures?return-to=https%3A%2F%2Fapp.circleci.com%2Fpipelines%2Fgithub%2Fdeepcharles%2Fruptures
I do not have the necessary right to proceed with it.

Reply:
Done